keel-sql/src/main/resources/db/databaseChangeLog.yml

```diff
@@ -20,3 +20,12 @@ databaseChangeLog:
   - include:
       file: changelog/20190804-create-application-veto-table.yml
       relativeToChangelogFile: true
+  - include:
+      file: changelog/20190724-environment-constraints.yml
```

Comment:
worth renaming these changelog files to have today's date?

Reply:
Ok, I did